core/common/src/annotation/TextStyle.ts

```diff
@@ -172,6 +222,17 @@ export interface TextStyleSettingsProps {
    * Default: {shape: "none", fill: "none", border: black, borderWeight: 1} for no frame.
    */
   frame?: TextFrameStyleProps;
+  /** The offset (in meters) from the left edge of the text block to the start of the line of text.
+   * In lists, the indentation is added to offset of list items.
+   * [[listMarker]]s will be centered on [[indentation]] + [[tabInterval]]*(depth - 1/2).
+   * [[Paragraph]]s will start at [[indentation]] + [[tabInterval]]*depth.
+   * Default: 0 meters.
+   */
+  indentation?: number;
+  /** The marker used to indicate the start of a list item.
+   * Default: "1.".
+   */
+  listMarker?: ListMarker;
```

Comment:
general question, is there a reason we default to an order list here? I guess in things like markdown or word, there isn't really a "default" maker for order lists (unordered lists I guess use defaults based on whether you're a sublist or not). Just curious if this is a design decision for a reason, cause I probably would've defaulted to an unordered list, or not allow value to be undefined or something. Or have a boolean isOrderedList somewhere that chooses the appropriate default for you (but that sounds annoying too)